net-vpn/vtun/vtun-3.0.3-r1.ebuild

```diff
@@ -14,32 +14,29 @@ SLOT="0"
 KEYWORDS="~alpha amd64 ppc ~sparc x86"
 IUSE="lzo socks5 ssl zlib"
 
-RDEPEND="ssl? ( dev-libs/openssl:0 )
+RDEPEND="
+	ssl? ( dev-libs/openssl:0 )
 	lzo? ( dev-libs/lzo:2 )
 	zlib? ( sys-libs/zlib )
 	socks5? ( net-proxy/dante )"
-DEPEND="${RDEPEND}
-	sys-devel/bison"
+DEPEND="${RDEPEND}"
+BDEPEND="sys-devel/bison"
 
 DOCS="ChangeLog Credits FAQ README README.Setup README.Shaper TODO"
-
 CONFIG_CHECK="~TUN"
 
+PATCHES=(
+	"${FILESDIR}"/${P}-includes.patch
+	"${FILESDIR}"/${PN}-3.0.2-remove-config-presence-check.patch
+	"${FILESDIR}"/${P}-gcc5.patch
+	"${FILESDIR}"/${PN}-libssl-1.1.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+)
+
 src_prepare() {
-	sed -i Makefile.in \
-		-e '/^LDFLAGS/s|=|+=|g' \
-		|| die "sed Makefile"
-	eapply "${FILESDIR}"/${P}-includes.patch
-	# remove unneeded checking for /etc/vtund.conf
```

Comment:
put these comments in the `PATCHES` array

Reply:
Done